README.md

```diff
@@ -32,18 +32,26 @@ This code is compiled into Wasm bytecode as part of the smart contract.
 - [cosmwasm-storage](https://github.com/CosmWasm/cosmwasm/tree/main/packages/storage) -
   A crate in this workspace. This optional addition to `cosmwasm-std` includes
   convenience helpers for interacting with storage.
+- [cw-storage-plus](https://github.com/CosmWasm/cosmwasm-plus/tree/main/packages/storage-plus) -
```

Comment:
multi-test is a powerful tool. Best not skip that.

Reply:
It's not ready for external consumption yet. I am still making some deeper API changes in it
With cosmwasm-plus 0.8.0 (or maybe 0.8.2 or 0.9.0) I will publish it.